src/app/products/charges/view-charge/view-charge.component.html

```diff
@@ -0,0 +1,122 @@
+<div fxLayout="row" fxLayoutAlign="end" fxLayoutGap="2%" fxLayout.lt-md="column" class="container m-b-20">
+  <button mat-raised-button color="primary">
+    <fa-icon icon="edit"></fa-icon>&nbsp;&nbsp;
+    Edit
+  </button>
+  <button mat-raised-button color="warn" (click)="deleteCharge()">
+    <fa-icon icon="trash"></fa-icon>&nbsp;&nbsp;
+    Delete
+  </button>
+</div>
+
+<div class="container">
+
+  <mat-card>
+
+    <mat-card-content>
+
+      <div fxLayout="row wrap" class="content">
+
+        <div fxFlex="50%" class="mat-body-strong">
+          Charge Type
+        </div>
+          
+        <div fxFlex="50%">
+          {{ chargeData.penalty === true ? 'Penalty' : 'Charge'}}
+        </div>
+
+        <div fxFlex="50%" class="mat-body-strong">
+          Currency
+        </div>
+
+        <div fxFlex="50%">
+          {{ chargeData.currency.name }}
+        </div>
+
+        <div fxFlex="50%" class="mat-body-strong">
+          Amount
+        </div>
+
+        <div fxFlex="50%">
+          {{ chargeData.amount }}
+        </div>
+
+        <div fxFlex="50%" class="mat-body-strong">
+          Charge Time Type
+        </div>
+
+        <div fxFlex="50%">
+          {{ chargeData.chargeTimeType.value }}
+        </div>
+
+        <div fxFlex="50%" class="mat-body-strong">
+          Charge Applies to
+        </div>
+
+        <div fxFlex="50%">
+          {{ chargeData.chargeAppliesTo.value }}
+        </div>
+
+        <div fxFlex="50%" class="mat-body-strong">
+          Charge Calculation Type
+        </div>
+
+        <div fxFlex="50%">
+          {{ chargeData.chargeCalculationType.value }}
+        </div>
+
+        <div fxFlex="50%" class="mat-body-strong">
+          Charge Payment Mode
+        </div>
+
+        <div fxFlex="50%">
+          {{ chargeData.chargePaymentMode.value }}
+        </div>
+
+        <div fxFlex="50%" class="mat-body-strong">
+          Active
+        </div>
+
+        <div fxFlex="50%">
+          {{ chargeData.active === true ? 'Yes' : 'No' }}
+        </div>
+
+        <div fxFlex="50%" class="mat-body-strong" *ngIf="chargeData.chargeTimeType.id == 9">
+          Fee Frequency
+        </div>
+
+        <div fxFlex="50%" *ngIf="chargeData.chargeTimeType.id == 9">
+          {{ chargeData.feeFrequency.value }}
+        </div>
+
+        <div fxFlex="50%" class="mat-body-strong" *ngIf="chargeData.chargeTimeType.id == 9">
+          Frequency Interval
+        </div>
+
+        <div fxFlex="50%" *ngIf="chargeData.chargeTimeType.id == 9">
+          {{ chargeData.feeInterval }}
+        </div>
+
+        <div fxFlex="50%" class="mat-body-strong" *ngIf="chargeData.incomeOrLiabilityAccount">
+          Income or Liability Account
```

Comment:
```suggestion
          Income from Charges
```
Update as per community app.